static/js/reducers/location.js

```diff
@@ -0,0 +1,22 @@
+// @flow
```

Comment:
Wondering about this file and its path. For one I'm not sure this contains 'reducers' in the Redux sense (functions that take the previous state and an action, and return the next state), so that's a bit confusing. I would also argue that `qsPartialTokenSelector`, `qsVerificationCodeSelector`, and `qsNextSelector` are auth-specific, so they should go into an auth lib file or something like that. Personally I would lean toward grouping the helpers for getting querystring param values in `static/js/lib/querystring.js` or something like that, and the auth-specific stuff in `static/js/lib/auth.js`